### DISCUSSION TITLE
3.3.3: Builds during 'install' target

### DISCUSSION THREAD

**Discussion by yurivict** [pawelsalawa/sqlitestudio#5429] [General]
### Details
It keeps building in the 'install' target: https://people.freebsd.org/~yuri/sqlitestudio-3.3.3.log
(after the ```Staging for sqlitestudio-3.3.3``` message)

Not sure if this is a regression, but this behavior is certainly wrong.

OS: FreeBSD 12.2

**Comment by pawelsalawa** [comment]
I'm not sure what the problem is. Could you put it in different words?

**Comment by yurivict** [comment]
The ```install``` target doesn't install SQLiteStudio, instead it compiles C++ files.

**Comment by pawelsalawa** [comment]
This is weird. There were no changes to makefiles/project files, except for the yml files used by GitHub actions only and for MacOS bundle builder script. Linux/Unix build was not touched.

I don't know why it behaves this way under FreeBSD. In any case, you can review all changes between 3.3.2 and 3.3.3 under this link https://github.com/pawelsalawa/sqlitestudio/compare/3.3.2...3.3.3 (there is a tab "Files changed" - very helpful).

**Comment by yurivict** [comment]
Bug in the cmake project: https://gitlab.kitware.com/cmake/cmake/-/issues/22094
